packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -726,6 +727,7 @@ class TimeInput extends React.Component {
               key="meridiem_select_box"
             >
               <select
+                aria-label={this.context.intl.formatMessage({ id: 'Meridiem Select' })}
```

Comment:
I would be more descriptive with this label, perhaps saying Select Meridiem instead of meridiem select. If that doesn't sound well, I would leave it as just Meridiem.

Reply:
Looks like we are still waiting for [translations](https://github.com/cerner/terra-core/blob/time-input-labels/packages/terra-time-input/translations/en.json).